waku/v2/node/wakunode2.go

```diff
@@ -882,6 +883,11 @@ func (w *WakuNode) Peers() ([]*Peer, error) {
 		}
 
 		addrs := w.host.Peerstore().Addrs(peerId)
+		hostInfo, _ := multiaddr.NewMultiaddr(fmt.Sprintf("/p2p/%s", peerId.Pretty()))
```

Comment:
Suggestion: Better to make this a multiaddr utility function that can be used across.